owoify/index.js

```diff
@@ -1,32 +1,104 @@
-import {webpackModules} from "@cumcord/modules";
-
-let unpatch;
+import { webpackModules } from "@cumcord/modules";
+function randomarray(array) {
+  return array[Math.floor(Math.random() * array.length)];
+}
+function random(min, max) {
+  return Math.round(Math.random() * (max - min) + min);
+}
 
-function random(array) {
-    return array[Math.floor(Math.random() * array.length)];
+function shuffle(array) {
+  let currentIndex = array.length,
+    randomIndex;
+  while (currentIndex != 0) {
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex--;
+    [array[currentIndex], array[randomIndex]] = [
+      array[randomIndex],
+      array[currentIndex],
+    ];
+  }
+  return array;
 }
 
+let output = "";
+let x = "";
+let ismodified = false;
 export default {
-    onLoad() {
-        unpatch = cumcord.patcher.after("sendMessage", webpackModules.findByProps("sendMessage"), (args) => {
-            if (args[1].content.startsWith("!owo")) {
-                const msg = args[1].content.replace("!owo", "");
+  onLoad() {
+    const endings = [
+      "owo",
+      "UwU",
+      ">w<",
+      "^w^",
+      "●w●",
+      "☆w☆",
+      "𝗨𝘄𝗨",
+      "(´꒳`)",
+      "♥(。U ω U。)",
+      "(˘ε˘)",
+      "( ˘ᴗ˘ )",
+      "(*ฅ́˘ฅ̀*)",
+      "*screams*",
+      "*twearks*",
+      "*sweats*",
+    ];
+    console.log("shit got loaded");
+    const removeCommand = cumcord.commands.addCommand({
+      name: "uwuify",
+
+      description: "uwuify your amazing string",
+      args: [
+        {
+          name: "string",
+          description: "the thing thats getting uwufified",
+          type: "string",
+          required: true,
+        },
+      ],
+
+      handler: (ctx, send) => {
+        output = "";
+        var input = ctx.args.string.split(" ");
+        for (let i = 0; i < input.length; i++) {
+          ismodified = false;
+          //yes its stutter but fuck you im too lazy to fix it
+          let chancesutter = random(1, 5);
+          x = input[i]
+            .toLowerCase()
+            .replace("r", "w")
+            .replace("l", "w")
+            .replace("ne", "nye")
+            .replace("no", "nyo")
+            .replace("ni", "nyi");
```

Comment:
You're not taking into consideration capital letters won't be replaced. See if you can do the same type of replacement for the other three `.replace` invocations.
```suggestion
            .replace(/[rl]/g, "w")
            .replace(/[RL]/g, "W")
            .replace("ne", "nye")
            .replace("no", "nyo")
            .replace("ni", "nyi");
```

Reply:
look at line 66, everything is lowercase anyway,

Reply:
Apologies, oversight!